controllers/game.controller.js

```diff
@@ -90,14 +91,23 @@ module.exports.updateGame = (req, res) => {
         const {title, genre, price} = req.body;
 
         const game = games.find(item => item.gameId === gameId);
-        if(title){
-            game.title = title;
-        }
-        if(genre){
-            game.genre = genre;
+
+        if(title.trim() === "" || genre.trim() == "" || isNaN(price) || price < 0){
+            res.json({
+                code: 500,
+                message: "title, genre or price is invalid"
+            })
         }
-        if(price){
-            game.price = price;
+        else{
+            if(title){
```

Comment:
There's no need to check if title, genre, and price have values again; this was already verified above